packages/web/src/pages/user/[uid].tsx

```diff
@@ -38,22 +41,47 @@ const UserProfilePage: NextPage = () => {
     fetchUser();
   }, [uid]);
 
+  React.useEffect(() => {
+    const checkUser = async () => {
+      try {
+        const res = await fetch('/api/users/me');
+
+        if (res.ok) {
+          const data = await res.json();
+
+          if (user && data.id == user.id) {
+            setIsCurrentUser(true);
+          }
+        }
+        setDidCheckCurrentUser(true);
+      } catch {}
```

Comment:
Be sure to set your error message if something in here throws by using `setErrorMessage`.

Reply:
Bump 👆